src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -63,25 +79,14 @@ class ManualColumnResize extends BasePlugin {
       return;
     }
 
-    this.manualColumnWidths = [];
-    const initialColumnWidth = this.hot.getSettings().manualColumnResize;
-    const loadedManualColumnWidths = this.loadManualColumnWidths();
+    this.columnWidthsMap = new ValueMap(() => void 0);
+    this.columnWidthsMap.addLocalHook('init', () => this.onMapInit());
+    this.columnIndexMapper.registerMap(COLUMN_WIDTHS_MAP_NAME, this.columnWidthsMap);
 
     this.addHook('modifyColWidth', (width, col) => this.onModifyColWidth(width, col));
     this.addHook('beforeStretchingColumnWidth', (stretchedWidth, column) => this.onBeforeStretchingColumnWidth(stretchedWidth, column));
     this.addHook('beforeColumnResize', (currentColumn, newSize, isDoubleClick) => this.onBeforeColumnResize(currentColumn, newSize, isDoubleClick));
 
-    if (typeof loadedManualColumnWidths !== 'undefined') {
-      this.manualColumnWidths = loadedManualColumnWidths;
-    } else if (Array.isArray(initialColumnWidth)) {
-      this.manualColumnWidths = initialColumnWidth;
-    } else {
-      this.manualColumnWidths = [];
-    }
-
-    // Handsontable.hooks.register('beforeColumnResize');
```

Comment:
Not really. I checked if they are registered somewhere by `Hook`'s singleton - they are not. Because of that, I decided to remove an unnecessary piece of code.